scripts/js/commands/generateObservabilityPage.test.ts

```diff
@@ -0,0 +1,78 @@
+// This code is a Qiskit project.
+//
+// (C) Copyright IBM 2025.
+//
+// This code is licensed under the Apache License, Version 2.0. You may
+// obtain a copy of this license in the LICENSE file in the root directory
+// of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+//
+// Any modifications or derivative works of this code must retain this
+// copyright notice, and modified files need to carry a notice indicating
+// that they have been altered from the originals.
+
+import { expect, test } from "@playwright/test";
+
+import {
+  extractEndpoints,
+  generateObservabilityTable,
+} from "./generateObservabilityPage";
```

Comment:
Awesome tests btw! You did a good job at making the input interesting enough to test but not too verbose.

This import is going to result in us unintentionally running the script when the test imports the file. That's because when you import the file, it will run the `main()` line at the very bottom of the file.

Instead, can you please move these helper functions to a new file `lib/observabilityPage.ts`? The test should also be in `lib`